style.css

```diff
@@ -22,3 +22,68 @@ body {
 }
 
 /* Add styles here! */
+
+/* Typography */
+h1 {
+  font-size: 2rem;
+  margin-block: 32px;
+}
+
+h2 {
+  font-size: 1.5rem;
+  margin-block: 24px;
+}
+
+p {
+  font-size: 1rem;
+  line-height: 1.5;
+  margin-block: 16px;
+}
+
+/* Layout */
+header {
+  padding-block: 64px;
+}
+
+main {
+  padding-block: 64px;
+  border-top: 8px solid var(--color-gray-medium);
+  background-color: var(--color-gray-light);
+}
+
+.max-width-wrapper {
+  max-width: 768px;
+  margin-inline: auto;
+}
+
+/* Elements */
+.intro-chunk {
+  max-width: 380px;
+  padding-inline: 24px;
+}
+
+.intro-chunk strong {
+  color: var(--color-primary);
+}
+
+.intro-chunk p:last-child {
+  margin-block-end: 0;
+}
+
+.card {
+  padding: 24px;
+  background-color: var(--color-white);
+  border-bottom: 8px solid var(--color-gray-medium);
+}
+
+.indented-heading {
+  padding: 16px 32px;
+  margin: -16px 0 0 -32px;
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches and the blocking or non-blocking decorations to further communicate your intent.  -->
**💬 suggestion:** ‏ Not a fan of shortcuts with no values not set.
https://csswizardry.com/2016/12/css-shorthand-syntax-considered-an-anti-pattern/
Maybe it would be better to use
```
margin-block-start: -16px;
margin-inline-start: -32px
```